src/panels/lovelace/hui-root.ts

```diff
@@ -815,7 +815,7 @@ class HUIRoot extends LitElement {
 
         .edit-mode app-header,
         .edit-mode app-toolbar {
-          background-color: var(--dark-color, #455a64);
+          background-color: var(--app-header-edit-background-color, var(--dark-color, #455a64));
```

Comment:
The `--dark-color` variable is only used here, we don't need yet another variable?

Reply:
I suggest you rename `dark-color` to `--app-header-edit-background-color`, and the same for the text color.

This should be removed:
https://github.com/home-assistant/frontend/blob/zwave-ozw-migration/src/panels/lovelace/hui-root.ts#L787-L788